sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -162,39 +162,38 @@ var data = new
 
 Response response = client.AnalyzeConversation(RequestContent.Create(data));
 
-using JsonDocument result = JsonDocument.Parse(response.ContentStream);
-JsonElement conversationalTaskResult = result.RootElement;
-JsonElement conversationPrediction = conversationalTaskResult.GetProperty("result").GetProperty("prediction");
+dynamic conversationalTaskResult = response.Content.ToDynamicFromJson();
```

Comment:
Per offline conversation, I think it would be good to pass `JsonPropertyNames.CamelCase` to indicate to the dynamic value that the JSON content uses camel case, so we can illustrate using the dynamic content with PascalCase names.  This makes the subsequent code substitutable for code that would be written if strongly-typed were later added to the library.

Reply:
I'll do that in a separate PR so it's easy to revert just that.

Reply:
Sounds good!